Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -278,22 +278,47 @@ void BlockFormattingContext::compute_width_for_floating_box(Box const& box, Layo
     if (margin_right.is_auto())
         margin_right = zero_value;
 
-    auto width = computed_values.width().has_value() ? computed_values.width()->resolved(box, width_of_containing_block_as_length).resolved(box) : CSS::Length::make_auto();
+    auto try_compute_width = [&](auto const& a_width) {
```

Comment:
"try" implies fallibility, but this can't fail...?

Reply:
Also, can we just pass by value here since a copy happens immediately anyway? Avoids the weird `a_width` name.